File: devcamp2018/app/src/main/java/com/psato/devfest/data/entity/ShowInfo.java
package com.psato.devfest.data.entity;

import com.google.gson.annotations.SerializedName;

/**
 * Created by psato on 29/10/16.
 */

public class ShowInfo {

    @SerializedName("type")
    private String mType;
    @SerializedName("score")
    private String mScore;
    @SerializedName("show")
    private Show mShow;

    public String getType() {
        return mType;
    }

    public String getScore() {
        return mScore;
    }

    public Show getShow() {
        return mShow;
    }
}

File: devcamp2018/app/src/main/java/com/psato/devcamp/data/entity/ShowInfo.kt
package com.psato.devcamp.data.entity

import com.google.gson.annotations.SerializedName

/**
 * Created by psato on 29/10/16.
 */

class ShowInfo {

    @SerializedName("type")
    var type: String? = null
    @SerializedName("score")
    var score: String? = null
    @SerializedName("show")
    var show: Show? = null
}
